api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -72,18 +75,42 @@ public Trace() {
     }
 
     /**
-     * @return the id
+     * @return the traceId
+     */
+    public String getTraceId() {
+        return traceId;
+    }
+
+    /**
+     * This method sets the trace id. When the fragment and trace id
```

Comment:
+1 for having the doc, no matter where :) But I agree with @pavolloffay  here: documenting the usage of `traceId` should be on the property definition, whereas particularities of the getter/setter should be on the javadoc for the method. Most of the time, the javadoc for the getter/setter will be, well, nothing :)